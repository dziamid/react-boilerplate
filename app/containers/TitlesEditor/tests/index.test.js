// import TitlesEditor from '../index';

import expect from 'expect';
// import { shallow } from 'enzyme';
// import React from 'react';

describe('<TitlesEditor />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
