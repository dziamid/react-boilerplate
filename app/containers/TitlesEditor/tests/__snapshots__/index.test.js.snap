exports[`<TitlesEditor /> should render with default props 1`] = `
<div>
  <HelmetWrapper
    meta={
      Array [
        Object {
          "content": "Description of TitlesEditor",
          "name": "description",
        },
      ]
    }
    title="TitlesEditor" />
  <div>
    <div>
      <H3>
        <FormattedMessage
          defaultMessage="Search Titles"
          id="app.containers.TitlesEditor.searchTitles"
          tagName="span"
          values={Object {}} />
        <span
          style={
            Object {
              "position": "absolute",
            }
          } />
      </H3>
      <Paper
        circle={false}
        rounded={true}
        transitionEnabled={true}
        zDepth={1}>
        <div>
          <Connect(ReduxForm)
            form="TitleEditor" />
          <FilterResults />
        </div>
      </Paper>
    </div>
    <div>
      <H3>
        <FormattedMessage
          defaultMessage="Edit Seniority And Relations"
          id="app.containers.TitlesEditor.editSeniorityAndRelations"
          tagName="span"
          values={Object {}} />
      </H3>
      <Paper
        circle={false}
        rounded={true}
        transitionEnabled={true}
        zDepth={1}>
        <div>
          <SingleTitleEditor
            onProximityChange={[Function]}
            relations={Array []}
            title={Object {}} />
        </div>
      </Paper>
    </div>
  </div>
</div>
`;
